client/src/app/page.module.scss

```diff
@@ -8,6 +8,19 @@
   padding: 6rem var(--side-padding);
 }
 
+.login {
+  width: 646px;
+  padding: 32px;
+  align-self: center;
+  align-items: stretch;
+
+  @media screen and (max-width: vars.$breakpoint-md) {
+    min-width: 343px; // Not working for some reason, never uses this. Reviewer please advise.
+    flex-shrink: 1;
+    width: 100%;
+  }
+}
+
```

Comment:
What's the min-width supposed to accomplish?

Reply:
Trying to get the login card to smoothly transition to the `min-width` size. Removing the `width: 100%` attribute makes the card not shrink (unless you add a `display flex` but then other problems happen)